### PR TITLE
LRAC-14401

### PR DESCRIPTION
**Pull request by riccardo-ferrari**

### CODE REVIEW

**Comment by riccardo-ferrari** [comment]
ci:test:sf

**Comment by riccardo-ferrari** [comment]
Hey @interaminense ,

Please review carefully!

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/experiments/pages/__tests__/ExperimentOverviewPage.tsx [hunk 1]
```diff
@@ -110,4 +110,29 @@ describe('ExperimentOverviewPage', () => {
...
 	});
+
+	it('renders publish and delete button to experiment to status FINISHED_WINNER', async () => {
+		const {container, getByRole} = render(
+			<WrappedComponent status='FINISHED_WINNER' />
+		);
+
+		await waitForLoadingToBeRemoved(container);
+
+		const reviewButton = getByRole('link', {
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/experiments/pages/__tests__/ExperimentOverviewPage.tsx:121]: Can we rename it to publishButton?

**Review by interaminense — CHANGES_REQUESTED** [review]
Thanks @riccardo-ferrari. Just a minor comment about variant name.

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/experiments/pages/__tests__/ExperimentOverviewPage.tsx (hunk 1 above)
Comment (riccardo-ferrari) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/experiments/pages/__tests__/ExperimentOverviewPage.tsx]: done!

**Comment by ViniLopes20** [comment]
Hey @riccardo-ferrari, the failure of `relevant` is not related to your modifications. The PR looks good to be forwarded. Thanks!

**Comment by interaminense** [comment]
ci:forward:force

**Comment by interaminense** [comment]
PR sent manually https://github.com/brianchandotcom/liferay-portal/pull/140961

**Pull request closed without merging**